app/views/pages/i-belong.html.erb

```diff
@@ -0,0 +1,119 @@
+                </div>
+                </div>
+                <section class="ncce-section govuk-grid-column-full">
+                    <h2 class="govuk-heading-m">
+                        I Belong Champions
+                    </h2>
+                    <p class="govuk-body">
+                        Join our champions initiative, which recognises teachers paving the way for girls in computer science and supports them to create wider change.
+                    </p>
+                    <%= link_to 'Become a champion',
+                    login_path,
+                    class: 'button govuk-button govuk-!-margin-bottom-0'
+                    %>
+                </section>
+            </div>
+            <div class="govuk-width-container">
+                <div class=" govuk-main-wrapper">
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-full white-bg govuk-!-padding-9">
+                            <h2 class="govuk-heading-m notice-component__title govuk-!-margin-bottom-7 bordered-card__title">Resources to support you</h2>
+                            <%= render NonBorderedCardsComponent.new(
+                                class_name: 'resource-cards',
+                                cards_per_row: 3,
+                                cards: [
+                                {
+                                    image_url: 'pages/i-belong/book_icon.png',
+                                    title: 'I Belong handbook',
+                                    text: 'Our evidence-informed handbook is key to developing your action plan for increasing uptake computer science by girls in your school.',
+                                    link: {
+                                    link_title: 'Coming soon',
+                                    link_url: '#',
+                                    }
+                                },
+                                {
+                                    image_url: 'pages/i-belong/faces.png',
+                                    title: 'I Belong in computer science posters',
+                                    text: "Printed 'I Belong in computer science' posters are available on request!​",
+                                    link: {
+                                    link_title: 'Log in to request',
+                                    link_url:'/login',
+                                    }
+                                },
+                                {
+                                    image_url: 'pages/i-belong/text_icon.png',
+                                    title: 'Gender Balance in computing research',
+                                    text: 'Read our reports from the Gender Balance in computing research project.',
+                                    link: {
+                                    link_title: 'Explore the research',
+                                    link_url: '/login',
```

Comment:
This doesn't look right. See https://github.com/NCCE/teachcomputing.org-issues/issues/2424 :
 
> Resources for support - this should be accessible upon enrolment to the program except GBIC research. The GBIC link in the About Us Navigation bar should be removed and linked here. The only change will be an update to the text in the ‘Programme at a glance’ to
>  ‘This research project has now concluded. For information about our new programme to support girls into computer science, visit our [I Belong](https://teachcomputing.org/I-belong) page.

Reply:
resolved too thank you

Reply:
I'll push a commit to feature flag this last sentence so it turns on at the same time as your I Belong page.